ksqldb-engine/src/main/java/io/confluent/ksql/util/QueryMetadataImpl.java

```diff
@@ -174,6 +175,7 @@ public void initialize() {
   ) {
     QueryError.Type errorType = Type.UNKNOWN;
     try {
+      QueryLogger.error("Query exception", this.statementString);
```

Comment:
This is not a very good error message. Something like

```
Uncaught exception in query {e} 
```

(pass the exception and make it clear it's an uncaught one)